src/db/models.py

```diff
@@ -1,24 +1,64 @@
 from . import connection
+import psycopg2.sql as sql
 
 class ApplicationsModel:
+    TableName = "applications"
+    Model = {}
+
+    @classmethod
+    def setModel(cls, fieldnames, sample_data):
+        for field in fieldnames:
+            cls.Model[field] = str # force everything into string, deal with on client
+        
+        cls.Model["score"] = int
+        
+        conn = connection.get_connection()
+        cursor = conn.cursor()
+        try:
+            cursor.execute('DROP TABLE IF EXISTS %s' % ApplicationsModel.TableName)
+            cursor.execute('CREATE TABLE %s ()' % ApplicationsModel.TableName)
+            
+            for name, data_type in cls.Model.items():
+                if data_type is str:
+                    postgre_type = "TEXT"
+                if data_type is int:
+                    postgre_type = "SMALLINT"
+                if data_type is float:
+                    postgre_type = "real"
+                cursor.execute('ALTER TABLE %s ADD COLUMN "%s" %s' % (cls.TableName, name, postgre_type))
```

Comment:
Do we need to wrap cls.TableName with sql.Identifier here?

Reply:
It's double quoted, https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS spec tells us that a delimited or quoted identifier can contain any character. So I'll add the double quote